test/unit_tests/autoqasm/test_operators.py

```diff
@@ -930,3 +930,52 @@ def test_list_ops():
         assert np.array_equal(c, [[2, 3, 4], [2, 3, 4]])
 
     assert test_list_ops.build().to_ir()
+
+
+class TestFloorDiv:
+    def test_integer_division_on_intvars(self):
+        @aq.main(num_qubits=2)
+        def main():
+            a = aq.IntVar(5)
+            b = aq.IntVar(2)
+            c = a // b  # noqa: F841
+
+        expected_ir = """OPENQASM 3.0;
+int[32] c;
+qubit[2] __qubits__;
+int[32] a = 5;
+int[32] b = 2;
+int[32] __int_2__;
+__int_2__ = a / b;
+c = __int_2__;"""
+        assert main.build().to_ir() == expected_ir
+
+    def test_integer_division_on_mixed_vars(self):
+        @aq.main(num_qubits=2)
+        def main():
+            a = aq.IntVar(5)
+            b = aq.FloatVar(2.3)
+            c = a // b  # noqa: F841
```

Comment:
To fix the missing code coverage I think you can just add another line here like:
```
d = b // a  # noqa: F841
```